beamline-environments/collection-dev/meta.yaml

```diff
@@ -0,0 +1,22 @@
+package:
+  name: collection-dev
+  version: '2015-03'
+# https://github.com/ericdill/conda-prescriptions/blob/add-dp-v0.2.2/beamline-environments/proposed-deployment-strategy.rst
+
+
+build:
+  number: 1
+
+requirements:
+  run:
+    - openpyxl
```

Comment:
bluesky version is going to have to be pinned in the stable collection metapackage